store/helper/helper.go

```diff
@@ -1139,31 +1140,43 @@ func GetTiFlashTableIDFromEndKey(endKey string) int64 {
 
 // ComputeTiFlashStatus is helper function for CollectTiFlashStatus.
 func ComputeTiFlashStatus(reader *bufio.Reader, regionReplica *map[int64]int) error {
-	ns, _, _ := reader.ReadLine()
-	n, err := strconv.ParseInt(string(ns), 10, 64)
+	b, err := ioutil.ReadAll(reader)
 	if err != nil {
 		return errors.Trace(err)
 	}
-	for i := int64(0); i < n; i++ {
-		rs, _, _ := reader.ReadLine()
```

Comment:
I think the original loop is more efficient. And you could workaround the truncation by using `rs, isPrefix, _ := reader.ReadLine()`.

Moreover, you could use `ReadBytes`, suggested by the official document:

> ReadLine is a low-level line-reading primitive. Most callers should use ReadBytes('\n') or ReadString('\n') instead or use a Scanner. `

Also, assert the return value of `ReadXXX`

Reply:
I double checked the server side, seems they will only return a text of two lines. 
So I think can switch to `ReadString`.